tests/io/vasp/test_outputs.py

```diff
@@ -1708,7 +1708,6 @@ def test_standard(self):
         b = np.array([np.cross(a[1, :], a[2, :]), np.cross(a[2, :], a[0, :]), np.cross(a[0, :], a[1, :])])
         b = 2 * np.pi * b / vol
 
-        assert wavecar.filename == f"{VASP_OUT_DIR}/WAVECAR.N2"
```

Comment:
I don't think it's quite necessary for `Wavecar` to carry a `filename` attribute (It's only used for unit test), is there any use case that I'm not aware of?

Reply:
Thanks!